packages/devtools/.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "webextensions": true
+  },
+  "extends": ["plugin:@nrwl/nx/react", "../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "rules": {
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }]
```

Comment:
I added override of `no-extraneous-dependencies` rule. Without it, I get error:
<img width="705" alt="image" src="https://user-images.githubusercontent.com/28751745/164010772-a78c8129-fabd-442b-9b6e-47f1caed6bab.png">

Reply:
```suggestion
```

We don't need need it, but we need it to specify `react` & `react-dom` to `dependencies` in `packages/devtools/package.json`:

```
    "react": "17.0.2",
    "react-dom": "17.0.2"
```